Mathlib/Algebra/Ring/NegOnePow.lean

```diff
@@ -70,6 +70,12 @@ lemma negOnePow_eq_neg_one_iff (n : ℤ) : n.negOnePow = -1 ↔ Odd n := by
     contradiction
   · exact negOnePow_odd n
 
+theorem negOnePow_natAbs (n : ℤ) : ((n.negOnePow : ℤ)).natAbs = 1 := Int.units_natAbs _
```

Comment:
Is this really necessary (instead of using `Int.units_natAbs`)?

Reply:
Well, I don't know since as you said it is essentially `Int.units_natAbs`. I guess one reason to keep it is that people would expect this result to exist and not realized that they could use `Int.units_natAbs`...

Reply:
`exact?` finds `Int.units_natAbs n.negOnePow` it, so I would think it can safely be left out.

Reply:
Ok. I'll remove it and also `Equiv.Perm.sign_natAbs` I guess.